src/components/composites/Button/index.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import { StyleSheet, TextStyle, TouchableWithoutFeedbackProps } from "react-native";
 import Ripple from "react-native-material-ripple";
 import styled from "styled-components";
+import { MaterialIcons } from "@expo/vector-icons";
```

Comment:
There should an empty line between import groups, I have added a new eslint rule to master. Please pull once from master and fix it. I'll merge it then.